src/main/java/mekanism/common/integration/OreDictManager.java

```diff
@@ -345,6 +344,12 @@ public static void init() {
 			InfuseRegistry.registerInfuseObject(ore, new InfuseObject(InfuseRegistry.get("DIAMOND"), 10));
 			RecipeHandler.addEnrichmentChamberRecipe(StackUtils.size(ore, 1), new ItemStack(Items.diamond));
 		}
+		
+		for(ItemStack ore : OreDictionary.getOres("gemDiamond"))
```

Comment:
gemDiamond compat for enrichment chamber needs to be removed as its already implemented from terralizationcompat further down.